drivers/soundwire/intel.c

```diff
@@ -338,37 +340,45 @@ static int intel_shim_init(struct sdw_intel *sdw)
 	/* Initialize Shim */
 	ioctl |= SDW_SHIM_IOCTL_BKE;
 	intel_writew(shim, SDW_SHIM_IOCTL(link_id), ioctl);
+	DELAY;
```

Comment:
this macro is just too vague. It's probably better to just add the explicit usleep_range directly.